### ISSUE TITLE
Another Easybytez problem

### ISSUE THREAD

**Issue by davboc** [pyload/pyload#2113]
```
127	25.10.2015 20:42:51	INFO	Download restarted: part09.rar | TEXTAREA F1 not found
126	25.10.2015 20:42:49	INFO	ADDON UnSkipOnFail: No duplicates found
125	25.10.2015 20:42:49	INFO	ADDON UnSkipOnFail: Looking for skipped duplicates of: part08.rev (pid:91)
124	25.10.2015 20:42:49	WARNING	Download failed: part08.rev | TEXTAREA F1 not found
123	25.10.2015 20:42:49	DEBUG	HOSTER XFileSharing[830]: WwwEasybytezCom | Config option `fallback` not found, use default `True`
122	25.10.2015 20:42:49	DEBUG	HOSTER XFileSharing[830]: WwwEasybytezCom | Getting download link #1...
121	25.10.2015 20:42:49	INFO	HOSTER XFileSharing[830]: WwwEasybytezCom | Processing as free download...
120	25.10.2015 20:42:49	DEBUG	HOSTER XFileSharing[830]: WwwEasybytezCom | LOAD URL http://www.easybytez.com/link/part08.rev | redirect=True | cookies=True | get={} | req=None | decode=True | multipart=False | post={} | ref=False | just_header=False
119	25.10.2015 20:42:49	INFO	HOSTER XFileSharing[830]: WwwEasybytezCom | Link status: online
118	25.10.2015 20:42:49	INFO	HOSTER XFileSharing[830]: WwwEasybytezCom | Link size: 251.00 MiB (263192576 bytes)
117	25.10.2015 20:42:49	INFO	HOSTER XFileSharing[830]: WwwEasybytezCom | Link name: part08.rev
116	25.10.2015 20:42:49	DEBUG	HOSTER XFileSharing[830]: WwwEasybytezCom | Previous link info: {'status': 2, 'url': u'http://www.easybytez.com/link/part08.rev', 'size': 263192576, 'name': u'part08.rev', 'pattern': {'S': u'251.0', 'U': u'MB', 'N': u'part08.rev'}}
115	25.10.2015 20:42:49	DEBUG	HOSTER XFileSharing[830]: WwwEasybytezCom | Link info: {'status': 2, 'name': u'part08.rev', 'url': u'http://www.easybytez.com/link/part08.rev', 'pattern': {'S': u'251.0', 'U': u'MB', 'N': u'part08.rev'}, 'size': 263192576}
114	25.10.2015 20:42:49	INFO	HOSTER XFileSharing[831]: WwwEasybytezCom | Waiting 1 seconds...
113	25.10.2015 20:42:49	DEBUG	HOSTER XFileSharing[831]: WwwEasybytezCom | RECONNECT not required | Previous wantReconnect: False
112	25.10.2015 20:42:49	DEBUG	HOSTER XFileSharing[831]: WwwEasybytezCom | WAIT set to timestamp 1445802171.487280 | Previous waitUntil: 1445802168.575202
111	25.10.2015 20:42:49	DEBUG	HOSTER XFileSharing[831]: WwwEasybytezCom | Getting download link #1...
110	25.10.2015 20:42:49	INFO	HOSTER XFileSharing[831]: WwwEasybytezCom | Processing as free download...
109	25.10.2015 20:42:49	INFO	HOSTER XFileSharing[830]: WwwEasybytezCom | Grabbing link info...
108	25.10.2015 20:42:49	DEBUG	HOSTER XFileSharing[830]: WwwEasybytezCom | Config option `use_premium` not found, use default `True`
107	25.10.2015 20:42:49	WARNING	HOSTER XFileSharing[830]: WwwEasybytezCom | Plugin may be unstable
106	25.10.2015 20:42:49	DEBUG	HOSTER XFileSharing[830]: WwwEasybytezCom | Plugin status: testing
105	25.10.2015 20:42:49	DEBUG	HOSTER XFileSharing[830]: WwwEasybytezCom | Plugin version: 0.62
104	25.10.2015 20:42:49	DEBUG	HOSTER XFileSharing[831]: WwwEasybytezCom | LOAD URL http://www.easybytez.com/link/part09.rar | redirect=True | cookies=True | get={} | req=None | decode=True | multipart=False | post={} | ref=False | just_header=False 
```

Pyload can't download nothing. In queue I see this error: "TEXTAREA F1 not found"

**Comment by GammaC0de** [comment]
this file is offline:

`File Not Found`

**Comment by davboc** [comment]
This isn't the real link, only an example 

Il 26 ottobre 2015 01:32:59 CET, GammaC0de <notifications@github.com> ha scritto:

**Comment by willoz88** [comment]
same for me with free account

**Comment by GammaC0de** [comment]
URL plz

**Comment by davboc** [comment]
@GammaC0de 

```
http://www.easybytez.com/zf26a2dujhx9/Jessabelle (2014) BDRip 576p.part05.rar
http://www.easybytez.com/7rh9ksw27owu/Jessabelle (2014) BDRip 576p.part05.rev
http://www.easybytez.com/cdgnxl11v836/Jessabelle (2014) BDRip 576p.part06.rar
http://www.easybytez.com/7nc9jxoaj406/Jessabelle (2014) BDRip 576p.part06.rev
```

**Comment by GammaC0de** [comment]
Ok, found the cause of this problem:
`UpdateManager` is again not in sync, please update manually from https://github.com/pyload/pyload/archive/stable.zip

**Comment by davboc** [comment]
@GammaC0de I can copy content of plugins folder into .pyload/userplugins?

**Comment by GammaC0de** [comment]
all the subfolders under `pyload/module/plugins` can be copied to `userplugins`

**Comment by davboc** [comment]
Yeah, now it works, thanks mate @GammaC0de 

There is the usual problem that pyload starts many downloads at same time but it can download only once http://i.imgur.com/xpG1paS.png

**Comment by GammaC0de** [comment]
Yes, I know, we already discussed that in #2105 and I asked vuolter to look for it but unfortunately, he will not have time for pyLoad until next week.
Meanwhile, you can make a temporary fix by editing `XFSHoster.py` and adding
```python
        super(XFileSharing, self).setup()
``` 
to line 62

**Comment by davboc** [comment]
Sorry, I did not understand that. Thanks again!

**Comment by GammaC0de** [comment]
Sorry, I made a mistake:
You should edit `XFileSharing.py` and add that line to line 41.
